homeassistant/components/openweathermap/sensor.py

```diff
@@ -161,6 +162,9 @@ async def async_setup_entry(
     name = domain_data.name
     weather_coordinator = domain_data.coordinator
 
+    if domain_data.mode == OWM_MODE_FREE_FORECAST:
```

Comment:
when use switch to forecast mode, current sensors are useless, how can i remove them properly?